### DISCUSSION TITLE
Update both changelog-chromie discord channel & the CC website with all tuning information

### DISCUSSION THREAD

**Discussion by andovcr** [chromiecraft/chromiecraft#6026] [Suggestions]
There have been multiple issues with communication and misinformation around custom changes to "tuning" of dungeon and raid content, often with these adjustments appearing in discord channels rather than "official documentation".  This leads to confusion amongst the player base of what content has what tuning.

Could you please add a section/page to the CC website denoting all areas that tuning has been applied, by what specific values / percentages, and on what date it goes to the live server?  I think this would greatly aid player comprehension when the staff are frustrated about people complaining about content is too easy or too hard, or which metrics have been adjusted over time.

I think this solves a knowledge gap that players want certainty on, while also removing stress from staff for having conversations in #general devolve into arguments where someone walks away upset.

**Comment by balleny** [comment]
can fully agree that its very annoying and intransparent to have this kinda relevant information being not accessible except for searching to messages in discord. a pinned post, a statistic or something similar would be very helpful - and could be combined with other baseline information like regular x2 experience etc.

_partly, this goes hand in hand with previous mentioning of intransparency and poorly communicated information or information about changes and adjustments (e.g. raid/feature releases, [closed-door discussions](https://github.com/chromiecraft/chromiecraft/discussions/4916), adjustment / delays of features)_

**Comment by Zelfie** [comment]
Honey — Yesterday at 22:22
"The numbers have all been published in the announcements. I'm sorry you missed them in summer."

Yet, i checked the announcements and couldn't actually find any numbers on the tuning.

**Reply by Zelfie** [reply]
-edit
I see most extra changes are posted in rates n changes, my bad.
